src/genomicsdb_processor.h

```diff
@@ -64,14 +64,11 @@ class ColumnarVariantCallProcessor : public GenomicsDBVariantCallProcessor {
                const std::vector<genomic_field_t>& genomic_fields);
   PyObject* construct_data_frame() {
     int dims = 1;
-    npy_intp sizes[1] = { static_cast<npy_intp>(m_rows.size()) };
+    npy_intp sizes[1] = { static_cast<npy_intp>(m_sample_names.size()) };
     PyObject *calls = PyDict_New();
-    PyDict_SetItem(calls, PyUnicode_FromString("Row"), PyArray_SimpleNewFromData(dims, sizes, NPY_INT64, m_rows.data()));
-    PyDict_SetItem(calls, PyUnicode_FromString("Col"), PyArray_SimpleNewFromData(dims, sizes, NPY_INT64, m_cols.data()));
     PyDict_SetItem(calls, PyUnicode_FromString("Sample"), PyArray_SimpleNewFromData(dims, sizes, NPY_OBJECT, m_sample_names.data()));
     PyDict_SetItem(calls, PyUnicode_FromString("CHROM"), PyArray_SimpleNewFromData(dims, sizes, NPY_OBJECT, m_chrom.data()));
```

Comment:
Does `CHROM` need to be `CHR`?

Reply:
It has been `CHROM` for a long time, will change it `CHR` to match what we have in the json output.